tools/scripts/used-headers.txt

```diff
@@ -0,0 +1,62 @@
+# QNX platform/ libc headers
+arpa/inet.h
```

Comment:
All QNX platform /libc headers should never be used in our C++ code base directly - except in platforms.
Our c++ code base should only include our platform files. If you could do this check we could avoid possible problems with other platforms..
Could you adjust the tests in that way?